### PR TITLE
Add docs for LinkCheckerApi integration

### PR DESCRIPTION
**Pull request by camillavk**

### CODE REVIEW

File: docs/link_checker_api_integration.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Link Checker API Integrations
+
+Links on an edition are checked using the LinkChecker API. Links are checked when the `Check Links` button is clicked on the edition show page. This fires off a request to the Link Checker API using the `/create_batch` endpoint. Once this has checked all the links present in the edition it makes a callback with a `batch_report` that gets saved as a `LinkCheckerApiReport`.
```
Comment (thomasleese) [inline docs/link_checker_api_integration.md]: Could we link to the documentation for the Link Checker API and the various models/endpoints mentioned here?

File: docs/link_checker_api_integration.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## LinkCheckerApiReports / LinkCheckerApiReportLinks
+
+| Field                 |
```
Comment (thomasleese) [inline docs/link_checker_api_integration.md]: I don't think it's necessary to list the fields here, just a few sentences explaining each model and what it's used for.

File: docs/link_checker_api_integration.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Link Checker API Integrations
+
+Links on an edition are checked using the LinkChecker API. Links are checked when the `Check Links` button is clicked on the edition show page. This fires off a request to the Link Checker API using the `/create_batch` endpoint. Once this has checked all the links present in the edition it makes a callback with a `batch_report` that gets saved as a `LinkCheckerApiReport`.
+
+## LinkCheckerApiReports / LinkCheckerApiReportLinks
```
Comment (thomasleese) [inline docs/link_checker_api_integration.md]: Probably worth splitting this up into something like:

```
## Models

### LinkCheckerApiReports

...

### LinkCheckerApiReportLinks
```

File: docs/link_checker_api_integration.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Link Checker API Integrations
+
+Links on an edition are checked using the [LinkChecker API](https://github.com/alphagov/link-checker-api/blob/master/docs/api.md). Links are checked when the `Check Links` button is clicked on the edition show page. This fires off a request to the Link Checker API using the [`/create_batch`](https://github.com/alphagov/link-checker-api/blob/master/docs/api.md#post-batch) endpoint. Once this has checked all the links present in the edition it makes a callback with a [`batch_report`](https://github.com/alphagov/link-checker-api/blob/master/docs/api.md#batchreport-entity) that gets saved as a `LinkCheckerApiReport`.
+
+## Models
+
+### LinkCheckerApiReports
+
+The `LinkCheckerApiReports` are returned by the API contain a [report](https://github.com/alphagov/link-checker-api/blob/master/docs/api.md#linkreport-entity) of all the links present in an edition.
```
Comment (bevanloon) [inline docs/link_checker_api_integration.md]: Could we remove the trailing space please?

File: docs/link_checker_api_integration.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+### LinkCheckerApiReports
+
+The `LinkCheckerApiReports` are returned by the API contain a [report](https://github.com/alphagov/link-checker-api/blob/master/docs/api.md#linkreport-entity) of all the links present in an edition. 
+
+### LinkCheckerApiReportLinks
+
+These are saved as `LinkCheckerApiReportLinks`. These contain [information](https://github.com/alphagov/link-checker-api/blob/master/docs/api.md#linkreport-entity) on the warnings or errors on the links, including a `suggested_fix`.
+
+The `LinkCheckerApiReport` is displayed on the admin page with a list of the broken links found on the edition.
```
Comment (bevanloon) [inline docs/link_checker_api_integration.md]: and here - also subsequent line breaks

**Comment by camillavk** [comment]
Closing this and adding the commit to https://github.com/alphagov/whitehall/pull/3557

**Pull request closed without merging**